fluent-plugin-out-http.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: fluent-plugin-out-http
+  version: 1.3.4
+  epoch: 0
+  description: This is the Fluentd output plugin for enabling http access to the container
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - ruby3.2-fluent-plugin-out-http=${{package.version}}-r${{package.epoch}}
+    runtime:
+      - ruby3.2-fluentd
+      - ruby3.2-json-jwt
+      - ruby3.2-multi_json
+      - ruby3.2-net-http-persistent
+      - ruby3.2-openid_connect-1.1.8
+      - ruby3.2-prometheus-client
+      - ruby3.2-rack-oauth2
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - ruby-3.2
+      - ruby-3.2-dev
+      - build-base
+      - busybox
+      - git
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 821f4468c2db1ddcfaad8520b1d9a5b33f5d6eda89a2fc931da735b3d42a6f44
+      uri: https://github.com/fluent-plugins-nursery/fluent-plugin-out-http/archive/refs/tags/${{package.version}}.tar.gz
+
+  - uses: ruby/unlock-spec
+
+  - uses: ruby/build
+    with:
+      gem: ${{package.name}}
+
+  - uses: ruby/install
+    with:
+      gem: ${{package.name}}
+      version: ${{package.version}}
+
+  - uses: ruby/clean
+
+update:
+  enabled: true
+  github:
+    identifier: fluent-plugins-nursery/fluent-plugin-out-http
```

Looks like it's defaulting to releases which the upstream repo doesn't have. Probably need to add this flag

```suggestion
    identifier: fluent-plugins-nursery/fluent-plugin-out-http
    use-tag: true
```